sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -90,7 +90,7 @@ export class BatchingReceiver extends MessageReceiver {
         receiver.session.removeListener(SessionEvents.sessionError, onSessionError);
 
         const sessionError = context.session && context.session.error;
-        let error = new MessagingError("An error occuured while receiving messages.");
+        let error = new MessagingError("An error occurred while receiving messages.");
```

Comment:
As discussed in other PRs, lets not change any existing error messages. While it is not advisable to program against error messages, there might be people who might be doing exactly that, and this change can break them.

Reply:
This was per @AlexGhiondea's comment - https://github.com/Azure/azure-sdk-for-js/pull/6004#issuecomment-552584831
Might be a minor typo that we might as well fix? Since we are likely moving away from hard dependency on error messages? cc @bterlson

Reply:
I think we should fix the typo personally.